packages/core/src/__tests__/embrace.test.ts

```diff
@@ -332,14 +338,21 @@ describe("Session Properties Tests", () => {
 
 describe("Payers Test", () => {
   test("setUserAsPayer", async () => {
-    await setUserAsPayer();
-    // TODO uncomment the expect once the method is implemented
-    // expect(mockSetUserAsPayer).toHaveBeenCalled();
+    const promiseToResolve = setUserAsPayer();
+    jest.runAllTimers();
+    const result = await promiseToResolve;
+
+    expect(mockSetUserAsPayer).toHaveBeenCalled();
+    expect(result).toBe(false);
```

Comment:
there's not much value in asserting this since it's just the value returned from the mock in this test file